src/azure-cli/azure/cli/command_modules/acs/agentpool_decorator.py

```diff
@@ -1519,6 +1519,20 @@ def get_if_none_match(self) -> str:
         """
         return self.raw_param.get("if_none_match")
 
+    def get_ignore_pod_disruption_budget(self) -> bool:
```

Comment:
Where is the value of this option passed?

Reply:
Seems the option is only used by command `aks nodepool delete`, if that's the case, you could undo your changes in this file as the codes here are only for `aks nodepool add/update` and `aks create/update`.